fedbiomed/common/constants.py

```diff
@@ -39,7 +40,16 @@
 JOB_PREFIX = 'job_'
 """Prefix for job ID"""
 
+
+ETC_FOLDER_NAME = 'etc'
```

Comment:
- `ETC_FOLDER_NAME` is duplicate of `CONFIG_FOLDER_NAME` => removing it
- `CERTS_FOLDER_NAME` could be also used for the existing hardcoded use of `etc/certs` => modifying this